WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -26,6 +25,7 @@ private static SQLiteDatabase getWritableDb() {
     public static void createTables(SQLiteDatabase db) {
         db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
                 + "person_id               INTEGER DEFAULT 0,"
+                + "site_id                 TEXT,"
```

Comment:
@maxme Yeap, it's used for wp.com sites only and as @hypest mentioned above, the question was for the remote id. So basically, whether to name it as `site_id` or `blog_id`? It'll still have the `local_blog_id` no matter what, because we need it to make sure we get the correct person when we query the DB. Hope that clears it.

@hypest I meant we use `SiteSettings` instead of `BlogSettings`, or `DeleteSite` instead of `DeleteBlog` etc.